docs/Install/HPE_Swarm_Learning_installation.md

```diff
@@ -2,6 +2,6 @@
 
 1.   [Download the License Server](Download_the_License_Server.md) 
 2.   [Install the License Server and download Swarm Learning](Install_the_License_Server.md) 
-3.   [Install HPE Swarm Learning](Install_HPE_Swarm_Learning.md) 
+3.   [Running SLM-UI Installer](Running_SLM-UI_Installer.md) 
```

Comment:
Not fully done ?  I had given a few lines to be added here in the original comment, right ?